Projects/FeatureInformation/Scene/InformationViewModel.swift

```diff
@@ -28,47 +30,53 @@ final class InformationViewModel: ObservableObject {
 	private var user: User = User.user1
 	
 	// detail api
+	@MainActor
 	func fetchMakHoly() {
-		
+		Task {
+			
+			do {
+				let makHoly = try await maHolyRepo.fetchDetail(makNumber: 5, userId: 1)
+				self.makHoly = makHoly
+				print("fetchMakHoly Completed : ------- \n \(makHoly) \n -------")
+				self.isFetchCompleted = true
+			} catch {
+				print("fetchMakHoly 오류")
```

Comment:
```suggestion
				Logger.debug(error: error, message: "")
```
p1
제가 추후에 네트워크 에러 로직들을 세세하게 추가할 예정인데, 우선은 이 에러 로직으로 통일할 예정이라 이 코드 추가해주시면 감사하겠습니다! 👍

Reply:
오씥! 넵넵